organization/_posts/2017-10-18-startup.md

```diff
@@ -0,0 +1,108 @@
+  * One room for 200 is the current booking
+    * However, it should be possible to get multiple rooms for up to 50 people instead
+  * Costs would be €100 for the two days (I believe that includes coffee and lunch though)
+  * We need to bear in mind the audience
+    * CHEP is more the experts in the field, not the fresh students
+  * Would some of the WGs have critical mass for a workshop/hackathon?
+  * Get different WGs together as a Birds of a Feather (e.g., frameworks and workload management)
+  * Could we offer some advanced tutorials? (Concurrency?)
+* If we decide to use this opportunity it would be useful to have a tentative plan in time for the second CHEP bulletin (15 November).
+
+* John - should take advantage of everyone together for a common discussion (that could be on Sunday PM allowing people to attend that session separately from other activities). 
+
+### Performance and Costs WG
+
+Graeme was in touch with Markus Schulz and will meet with him to discuss the HSF’s involvement.
+
+## CWP
+
+- #### General matters and Editorial Board
+  - Very significant progress
+    - Introduction and challenges sections are in good shape
+    - Almost all of the individual CWPs are now integrated
+    - Conclusions are in progress
+  - However, still not quite ready to release the first draft today, should happen tomorrow instead
+  - We will release the document in a stable form, so that people have a definite period on which to comment.
+  - Some numbers in S2I2 document might be wrong (CMS tape) - we need to be consistent, so Pete should cross-check
+  - Reminded that we want to have many people sign this document to show their support (different from individual CWPs)
+
+- #### Simulation
+  - No updates.
+
+- #### Machine Learning
+  - No news; for the synthesis paper used the summary from the S2I2 document.
+
+- #### Software Development
+  - Added author list (distinguished between editors and contributors). Still references TODO, then goes to arXiv.
+
+- #### Software Trigger and Event Reconstruction
+  - Last loose ends before arXiving.
+
+- #### Data Access, Organisation and Management
+  - No news.
+
+- #### Data and Software Preservation
+  - Section on current practice (from the LHC experiments) was added by Mike. Some review comments from DPHEP group being addressed.
+
+- #### Data Analysis and Interpretation
+  - Oli distributed draft for last comments to the general CWP list.
+
+- #### Workflow and Resource Management
+  - No news.
+
+- #### Visualization
+  - Riccardo is working on the last pieces in time for incorporation into the synthesis paper.
+
+- #### Event/Data Processing Frameworks
+  - Have been adding descriptions and goals. Ready for further review later today. Would like more on 5 year goals. Did incorporate some inputs from ALICE, Belle II, etc., but indirectly. Still open questions that need to be addressed on the roadmap (single node vs. multiple nodes).
```

Comment:
"but indirectly" I would reword as "incorporated previous high-level inputs from...", i.e. no contribution to details unfortunately.
The open question about single vs. multiple nodes is in the roadmap already. I just brought it up as an example where ALICE vs. the other approaches is important